frontend/src/scenes/cohorts/addPersonToCohortModalLogic.ts

```diff
@@ -0,0 +1,116 @@
+        setCohortUpdateLoading: (personId: string, loading: boolean) => ({ personId, loading }),
+    }),
+    reducers({
+        query: [
+            {
+                kind: NodeKind.DataTableNode,
+                source: {
+                    kind: NodeKind.ActorsQuery,
+                    fixedProperties: [],
+                    select: ['id', 'person_display_name -- Person'],
+                },
+                showPropertyFilter: false,
+                showEventFilter: false,
+                showExport: false,
+                showSearch: true,
+                showActions: false,
+                showElapsedTime: false,
+                showTimings: false,
+            } as DataTableNode,
+            {
+                setQuery: (state, { query }) => (isDataTableNode(query) ? query : state),
+            },
+        ],
+        addPersonToCohortModalVisible: [
+            false,
+            {
+                showAddPersonToCohortModal: () => true,
+                hideAddPersonToCohortModal: () => false,
+            },
+        ],
+        cohortUpdatesInProgress: [
+            {} as Record<string, boolean>,
+            {
+                setCohortUpdateLoading: (state, { personId, loading }) => ({
+                    ...state,
+                    [personId]: loading,
+                }),
+            },
+        ],
+    }),
+    loaders(({ props, values }) => ({
+        cohortPersons: [
+            { next: null, previous: null, results: [] } as PaginatedResponse<PersonType>,
+            {
+                loadCohortPersons: async () => {
+                    if (props.id == null || props.id === 'new') {
+                        return values.cohortPersons
+                    }
+                    const result = await api.cohorts.getCohortPersons(props.id)
+
+                    return result
+                },
+            },
+        ],
+    })),
+    listeners(({ props, actions }) => ({
+        addPersonToCohort: async ({ id }) => {
+            actions.setCohortUpdateLoading(id, true)
+            const cohortId = props.id
+            if (cohortId == null || cohortId === 'new') {
+                return
+            }
+            try {
+                const response = await api.cohorts.addPersonsToStaticCohort(cohortId, [id])
+                await actions.loadCohortPersons()
+                if (response) {
+                    lemonToast.success('Person added to cohort')
+                    const mountedCohortEditLogic = cohortEditLogic.findMounted({ id: cohortId })
+                    await mountedCohortEditLogic?.actions.updateCohortCount()
+
+                    const mountedDataNodeLogic = dataNodeLogic.findMounted({
+                        key: createCohortDataNodeLogicKey(cohortId),
+                    })
+                    mountedDataNodeLogic?.actions.loadData('force_blocking')
+                }
+            } catch {
+                lemonToast.error('Unable to add person to cohort')
```

Comment:
We don't want to swallow errors. It'll make it difficult to support.

```suggestion
            } catch (error) {
                console.error('Failed to add person to cohort:', error)
                lemonToast.error('Unable to add person to cohort')
```